### ISSUE TITLE
The Final Equation of Loss 

### ISSUE THREAD

**Issue by moegi161** [lc82111/Active-Contour-Loss-pytorch#4]
I wonder why you return  ` loss =  weight*lenth + region ` instead of  `lenth + weight*region` as the paper have written.
